pkgs/applications/misc/toot/default.nix

```diff
@@ -2,16 +2,16 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "toot";
-  version = "0.34.1";
+  version = "0.36.0";
 
   src = fetchFromGitHub {
     owner  = "ihabunek";
     repo   = "toot";
     rev = "refs/tags/${version}";
-    sha256 = "sha256-5LTd3FPodYxMm4zZJsAfO0O1Y0AXUxaz+ZtEh6b5Etw=";
+    sha256 = "sha256-gEQA9PASSKAMqulOaK8ynBXX7BdptY1uwdS1tOf3/Jc=";
   };
 
-  nativeCheckInputs = with python3Packages; [ pytest ];
+  checkInputs = with python3Packages; [ pytest ];
```

Comment:
this seems to break the build

Reply:
Thank you for your feedback.

On my system, I get this same error when using `nativeCheckInputs` and switching to `checkInputs` fixes it. I use the package in an overlay.

I will change this back to `nativeCheckInputs` and see what happens.

BTW is there a way I can test changes on the build system without modifying my PR or is it the recommended way?

Reply:
You can build it like `nix-build -A toot` and run it with `result/bin/toot`, testing with overlays is not the best idea